### DISCUSSION TITLE
Notesnook is now fully open source!

### DISCUSSION THREAD

**Discussion by thecodrr** [streetwriters/notesnook#857] [Announcements]
We worked all over the Notesnook codebase to finally bring to you a fully, 100% open source & end-to-end encrypted note taking app. If Notesnook was awesome before, it's even more awesome now.

Thank you to everyone who helped & supported us on this journey! We couldn't have done this without you guys.

**Comment by navilg** [comment]
is there any plan for open sourcing sync server or allow self-hosting ?

**Reply by dislazy** [reply]
This is the most important

**Reply by thecodrr** [reply]
Definitely. See our roadmap (https://notesnook.com/roadmap).

**Reply by yangyangyangxp123** [reply]
Hello! What's the problem with my Android APK always failing to build? Can you help me? Thank you!

**Comment by Bmswad1** [comment]
Congratulations 🥳🥳🥳

**Comment by shuvashish76** [comment]
It's FOSS review: https://news.itsfoss.com/notesnook-goes-open-source/

**Comment by trymeouteh** [comment]
Thank you! Looking forward to seeing the server being open sourced in the future also.
